### PR TITLE
COMDOX-818: 2.4.7-beta3 BIC highlights

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) adds backward-incompatible change highlights for 2.4.7-beta3.

## Affected pages

- https://developer.adobe.com/commerce/php/development/backward-incompatible-changes/highlights/

### CODE REVIEW

**Comment by jeff-matthews** [comment]
Dead link errors in PR validation are false positives and can be ignored. I'll try clearing the actions cache.

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 1]
```diff
@@ -9,6 +9,46 @@ keywords:
...
+
+The following module is affected by this change:
+
+* [Magento_SalesRule](https://developer.adobe.com/commerce/php/module-reference/module-sales-rule/)
+
+### New method and an optional parameter for multicoupons
+
+The following changes were introduced to implement the multicoupon functionalityin the [SalesRule](https://developer.adobe.com/commerce/php/module-reference/module-sales-rule/) module:
+
+* Optional parameter added to  Magento\SalesRule\Model\ResourceModel\Rule\Collection::setValidationFilter
```
Comment (keharper) [inline src/pages/development/backward-incompatible-changes/highlights.md:40]:
```suggestion
* Optional parameter added to  `Magento\SalesRule\Model\ResourceModel\Rule\Collection::setValidationFilter`
```

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 2]
```diff
@@ -9,6 +9,46 @@ keywords:
...
+* New method introduced: Magento\SalesRule\Model\Validator::initFromQuote
```
Comment (keharper) [inline src/pages/development/backward-incompatible-changes/highlights.md:41]:
```suggestion
* New method introduced: `Magento\SalesRule\Model\Validator::initFromQuote`
```

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 3]
```diff
@@ -9,6 +9,46 @@ keywords:
...
+
+All changes have been done in a way to minimize any impact to extensions and customizations, however, there are risks of conflict if an extension or customization extends the following:
```
Comment (keharper) [inline src/pages/development/backward-incompatible-changes/highlights.md:43]:
```suggestion
All changes have been done in a way to minimize any impact to extensions and customizations. However, there are risks of conflict if an extension or customization extends the following:
```

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 4]
```diff
@@ -9,6 +9,46 @@ keywords:
...
+
+* `Magento\SalesRule\Model\ResourceModel\Rule\Collection::setValidationFilter` and adds a parameter to this method (highly unlikely)
```
Comment (keharper) [inline src/pages/development/backward-incompatible-changes/highlights.md:45]:
```suggestion
* `Magento\SalesRule\Model\ResourceModel\Rule\Collection::setValidationFilter` and adds a parameter to this method.
```

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 5]
```diff
@@ -9,6 +9,46 @@ keywords:
...
+* `Magento\SalesRule\Model\Validator` and introduces a method with the same name `initFromQuote` (highly unlikely).
```
Comment (keharper) [inline src/pages/development/backward-incompatible-changes/highlights.md:46]:
```suggestion
* `Magento\SalesRule\Model\Validator` and introduces a method with the same name `initFromQuote`.
```

**Comment by jeff-matthews** [comment]
Thanks for graciously cleaning up my carelessness @keharper 😬

File: src/pages/development/backward-incompatible-changes/highlights.md [hunk 6]
```diff
@@ -9,6 +9,46 @@ keywords:
...
+* New system configuration for limiting coupon generation
+* New method and an optional parameter for multicoupons
+
+### Fixes to resolve compatibility issues with Symfony
+
+The latest Symfony 6.4 LTS is not supported, so changes were made to keep Symfony LTS 5.4 in 2.4.7-beta3. Extension developers must define strict typing for return values in classes that use the changed interface: `Magento\Framework\Console\Cli::getDefaultCommands`.
+
+The following module is affected by this change:
+
+*  Magento_Framework
```
Comment (keharper) [inline src/pages/development/backward-incompatible-changes/highlights.md:26]: This is not a module. You gave the path to the affected interface, so I think you can delete lines 24-27. (Sorry I missed that on the first pass.)

**Review by chernenm — APPROVED** [review]

**Review by keharper — APPROVED** [review]

**Pull request merged**